### ISSUE TITLE
Flat menu style not working

### ISSUE THREAD

**Issue by subnut** [coldfix/udiskie#270]
Running `udiskie -m flat` still results in a nested menu style instead of a flat menu style

**Comment by coldfix** [comment]
Hi,

the flat/nested menu style is only applied to the left-click menu currently. I think that command line option needs to be changed, or removed in favor of configuration.

**Comment by coldfix** [comment]
Yes, on appindicator only one type of menu can be shown, which was so far been chosen to be nested one. This is probably be confusing and should be changed according to command line options.
